pages/textbook-beta/summer-school/quantum-computing-and-quantum-learning-2021.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+  <main class="summer-school-page">
+    <TextbookBetaContentMenuSection
+      class="summer-school-page__content-menu"
+    />
+    <AppPageHeaderWithImage :cta="startLearningCTA" :back-link="backToTextbookHomeLink">
+      <template slot="title">
+        {{ headerTitle }}
+      </template>
+      <template slot="description">
+        <p
+          v-for="(paragraph, index) in headerDescription"
+          :key="index"
+          v-text="paragraph"
+        />
+      </template>
+      <template slot="image">
+        <img class="app-page-header-with-img__image" :src="headerImg">
+      </template>
+    </AppPageHeaderWithImage>
+    <PrerequisiteMaterialSection class="summer-school-page__section" :data="summerSchool2021Prerequisites" />
+    <ExternalRecommendedReadings class="summer-school-page__section" :links="links" />
+    <CoursePagesSection class="summer-school-page__section" :courses="courses" :img-base="imageUrlBase" />
+  </main>
+</template>
+
+<script lang="ts">
+import { Component } from 'vue-property-decorator'
+import QiskitPage from '~/components/logic/QiskitPage.vue'
+import { CtaClickedEventProp } from '~/constants/segment'
+import { GeneralLink, summerSchoolCourseStartLearningUrl } from '~/constants/appLinks'
+
+interface Course {
```

Comment:
you already have this interface on the other page. It could be a nice idea to move it to another file and export it from there. Maybe you can follow the same you did with the `Prerequisite` interface until we have a better solution

Reply:
Great suggestion, thanks!